library/core/functions.general.php

```diff
@@ -1740,7 +1740,11 @@ function getAllMentions($str) {
                 $parts2 = preg_split('`([\s.,;?!:])`', $part, 2, PREG_SPLIT_DELIM_CAPTURE);
                 $mention = $parts2[0];
             }
-            $mentions[] = $mention;
+
+            // Filter empty mentions
+            if ($mention !== "") {
```

Comment:
Since initial default value of `$mention = false` probably we need to:
```suggestion
            if (!empty($mention)) {
```
or just
```suggestion
            if ($mention) {
```